docs/guide/snaps-development-guide.md

```diff
@@ -0,0 +1,41 @@
+# Snaps Development Guide
+
+::: warning 🛠️ Under Construction 🛠️
+This page is under construction.
+:::
+
```

Comment:
I'm guessing that this won't still be under construction when this gets merged

```suggestion
```

Reply:
I see multiple references to this being under construction in the text of this page. Hmm. This doesn't seem ready.

We should be able to write a basic development guide that uses our snap template and the snap CLI.

Reply:
The "snaps tutorial" part of the intro could be adapted into a decent "development guide". It's not far off.

Reply:
That sounds like what I intended from the beginning, so--a rose by any other name!